### ISSUE TITLE
SourceMap warnings when launch React App CRA

### ISSUE THREAD

**Issue by jtwigg** [mobxjs/serializr#162]
With only 1 usage of 
```
import { serializeAll} from "serializr"

@serializeAll
export class CinemaState {
...
```

I'm getting compile errors.
NOTE: running `tsc` works, its when I run `DISABLE_ESLINT_PLUGIN=true react-scripts start` that I get these errors.

Sure enough, there is no `node_modules/serializr/src/` folder. 
Any ideas?

```
Compiled with warnings.

Failed to parse source map from '/Users/johntwigg/development//ui/node_modules/serializr/src/api/createModelSchema.ts' file: Error: ENOENT: no such file or directory, open '/Users/johntwigg/development//ui/node_modules/serializr/src/api/createModelSchema.ts'

<similar> x 20

WARNING in ./node_modules/serializr/lib/serializr.es.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '/Users/johntwigg/development//ui/node_modules/serializr/src/utils/invariant.ts' file: Error: ENOENT: no such file or directory, open '/Users/johntwigg/development//ui/node_modules/serializr/src/utils/invariant.ts'

<similar> x 20
...

**Comment by jtwigg** [comment]
@mweststrate 
https://github.com/mobxjs/serializr/pull/177

**Comment by NaridaL** [comment]
closed with #177
